pallets/phala/src/stakepool.rs

```diff
@@ -613,6 +666,38 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(0)]
+		pub fn check_and_maybe_force_withdraw(
+			origin: OriginFor<T>,
+			pid: u64,
+		)  -> DispatchResult {
+			ensure_signed(origin)?;
+			let now = <T as registry::Config>::UnixTime::now()
+				.as_secs()
+				.saturated_into::<u64>();
+			let mut pool = Self::ensure_pool(pid).expect("Stake pool must exist; qed.");
```

Comment:
`pid` is specified by the user. Must handle the returned `Err`, rather than unwrap it with `expect`